### ISSUE TITLE
Quality warning is not accessible

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#7678]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

Moved over from https://github.com/nextcloud/spreed/pull/7676

- [ ] Content can not be reached via keyboard
- [ ] Warning is not read out by screenreaders (should it be a showError instead, or just receive aria-live)
